modules/site-health/admin-site-health.php

```diff
@@ -172,6 +187,43 @@ public function info_options( $debug_info ) {
 		return $debug_info;
 	}
 
+	/**
+	 * Get an array with post_type as key and post IDs as value
+	 *
+	 * @param int $limit  Nb of post max to show per post type.
+	 * @return array
+	 *
+	 * @since   3.0
+	 */
+	public function get_post_ids_without_lang( $limit = 10 ) {
```

Comment:
Why not set the default value to 5 as it is called with this value in `info_options()` method ?

Reply:
Yes you're right